File: Java/Advanced Conditional Statements/Exercise/ACS Exercise/src/NewHouse.java
import java.util.Scanner;

public class NewHouse {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String typeFlowers = input.nextLine(); //"Roses", "Dahlias", "Tulips", "Narcissus", "Gladiolus"
        int amount = Integer.parseInt(input.nextLine());
        int budget = Integer.parseInt(input.nextLine());

        double total = 0;

        //prices
        double roses = 5, dahlias = 3.80, tulips = 2.80, narcissus = 3, gladiolus = 2.50;

        switch(typeFlowers){
            case "Roses":
                total = roses * amount;

                if(amount > 80){
                    total -= total * 0.1;
                }
                break;
            case "Dahlias":
                total = dahlias * amount;

                if(amount > 90){
                    total -= total* 0.15;
                }
                break;
            case "Tulips":
                total = tulips * amount;

                if(amount > 80){
                    total -= total * 0.15;
                }
                break;
            case "Narcissus":
                total = narcissus * amount;

                if(amount < 120){
                    total += total * 0.15;
                }
                break;
            case "Gladiolus":
                total = gladiolus * amount;

                if(amount < 80){
                    total += total * 0.2;
                }
                break;
        }

        if(total <= budget){
            System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", amount, typeFlowers, (double)budget - total);
        }else {
            System.out.printf("Not enough money, you need %.2f leva more.", total - budget);
        }
    }
}

File: Python/Basics/Conditional Statements Advanced/Exercise/New House.py
flowerType = input()
flowersAmount = int(input())
budget = int(input())
costs = 0
prices = [5, 3.80, 2.80, 3, 2.50] # rose dahlia tulip narcissus gladiolus

if flowerType == "Roses":
    costs = flowersAmount * prices[0]

    if flowersAmount > 80:
        costs -= costs * 0.1

elif flowerType == "Dahlias":
    costs = flowersAmount * prices[1]

    if flowersAmount > 90:
        costs -= costs * 0.15

elif flowerType == "Tulips":
    costs = flowersAmount * prices[2]

    if flowersAmount > 80:
        costs -= costs * 0.15

elif flowerType == "Narcissus":
    costs = flowersAmount * prices[3]

    if flowersAmount < 120:
        costs += costs * 0.15
elif flowerType == "Gladiolus":
    costs = flowersAmount * prices[4]

    if flowersAmount < 80:
        costs += costs * 0.2

if costs <= budget:
    print(f"Hey, you have a great garden with {flowersAmount} {flowerType} and {budget - costs:.2f} leva left.")
else:
    print(f"Not enough money, you need {costs - budget:.2f} leva more.")